03_launch_mgmt_cluster.sh

```diff
@@ -39,14 +36,31 @@ source lib/ironic_basic_auth.sh
 # BMO and Ironic deployment functions
 # ------------------------------------
 
+function deployIronicAndOrBMO() {
+  export NAMEPREFIX=${NAMEPREFIX:-"baremetal-operator"}
+
+  IRONIC_DATA_DIR="${IRONIC_DATA_DIR:-/opt/metal3/ironic/}"
+  IRONIC_AUTH_DIR="${IRONIC_AUTH_DIR:-${IRONIC_DATA_DIR}auth/}"
```

Comment:
```suggestion
  IRONIC_AUTH_DIR="${IRONIC_AUTH_DIR:-${IRONIC_DATA_DIR}/auth}"
```
Always expect directories not to have trailing slash. Shell doesn't care if you have multiple, but in case its missing, the AUTH_DIR would be invalid.